src/react-hook/react-hook.tsx

```diff
@@ -13,17 +13,15 @@ export const ModuleNameContext = React.createContext(null);
 let config;
 export function useConfig() {
   const moduleName = React.useContext(ModuleNameContext);
-  //  const [config, setConfig]: [null | Object, Function] = React.useState(null);
   if (!moduleName) {
     throw Error(
       "ModuleNameContext has not been provided. This should come from openmrs-react-root-decorator"
     );
   }
-  if (!config) {
+  if (config == null) {
```

Comment:
It is generally considered poor practice to use `==` instead of `===`, since type coercion in javascript is very tricky and can result in some weird bugs.

I do not think we should change this line of code at all. The falsiness check that is in place on the master branch seems best to me. Is there a falsy value that would be problematic to go into the if statement for?

```suggestion
  if (!config) {
```